opensrp-anc/src/main/assets/config/contact-summary.yml

```diff
@@ -1,5 +1,5 @@
 ---
-properties_file_name: "contact_summary"
+propertiesFileName: "contact_summary"
```

Comment:
Why are we changing the naming convention here?

Reply:
Reverted